modules/reference/pages/glossary.adoc

```diff
@@ -20,11 +34,21 @@ An internal TigerGraph characteristic of a TigerGraph cluster that determines ho
 
 A characteristic of the cluster design that will determine the number of copies of data that will be stored in the cluster.
 
+== resizing
+
+Allocating more or fewer resources to existing instances, enabling a balance between cost and performance as workloads change.
+
+== REST++/RESTPP
+
+A REST server customized for TigerGraph users.
 
 == role-based access control
 
 A policy-neutral access-control mechanism defined around roles and privileges.
 
+== schema
+
+A formal description of the graph. The schema defines what types of edges and vertices exist in a graph as well as their relationships.
```

Comment:
I would say "A formal description of a graph's data model" or "graph's architecture". If you just say "graph", some readers will be thinking of a particular set of data, which is not what we mean.